app/controllers/dashboard/dashboard-activity-controller.js

```diff
@@ -120,36 +135,36 @@ module.exports = (req, res) => {
       }))
     }
 
-    const namespace = getNamespace(clsXrayConfig.nameSpaceName)
-    const clsSegment = namespace.get(clsXrayConfig.segmentKeyName)
-
-    AWSXRay.captureAsyncFunc('ledgerClient_getTransactionSummary', function (subsegment) {
-      LedgerClient.transactionSummary(gatewayAccountId, fromDateTime, toDateTime, { correlationId: correlationId })
-        .then(result => {
-          subsegment.close()
-          response(req, res, 'dashboard/index', Object.assign(model, {
-            activity: result,
-            successfulTransactionsState: 'payment-success',
-            fromDateTime,
-            toDateTime,
-            transactionsPeriodString
-          }))
-        })
-        .catch((error, ledgerResponse) => {
-          subsegment.close(error)
-          const status = _.get(ledgerResponse, 'statusCode', 404)
-          logger.error(`[${correlationId}] Calling ledger to get transactions summary failed`, {
-            service: 'ledger',
-            method: 'GET',
-            status,
-            error
-          })
-          res.status(status)
-          response(req, res, 'dashboard/index', Object.assign(model, {
-            activityError: true
-          }))
+    const stripeAccountSetup = await getStripeAccountSetup(req.account, correlationId)
+    if (stripeAccountSetup) {
+      Object.assign(model, {
+        stripeAccountSetup: stripeAccountSetup
+      })
+    }
+
+    LedgerClient.transactionSummary(gatewayAccountId, fromDateTime, toDateTime, { correlationId: correlationId })
+      .then(result => {
```

Comment:
reckon we should use `await` here to be consistent within the function?

Reply:
Agree! left it so there won't be much refactoring in this PR. Will raise a separate PR or include in next PR (if small)